recipe/meta.yaml

```diff
@@ -8,16 +8,17 @@ package:
 source:
   fn: {{ version }}.zip
   url: https://github.com/casadi/casadi/releases/download/{{ version }}/casadi-source-v{{ version }}.zip
-  sha256: 63d2c752d819f1790636f2015990c83bade6dd425af91b4239518312f6f6a052
+  sha256: 559a850620094246ec7cd8cfe00d9d9214706687d5b9aba63e68c21afb60a5ed
   patches:
-    - patches/2965.patch
-    - patches/rtld_deepbind_null_environ_workaround.patch
-    - patches/0001-Modernize-Python-find_package.patch
-    - patches/0003-Bypass-CMake-crosscompiling-conda-forge-CI-issue.patch
+    - patches/0001-Rename-qpOASES-namespace-to-casadi_qpOASES.patch
+    - patches/0002-Work-around-RTLD_DEEPBIND-null-environ.patch
+    # - patches/0002-Export-difference-function.patch # currently not applied
```

Comment:
If it is not used, perhaps we can just rename it?

Reply:
Addressed in 7321d38c370af89a653f5f9bfd034ec2d76b1aa0.